extensions/amp-story/1.0/page-advancement.js

```diff
@@ -493,13 +495,18 @@ class ManualAdvancement extends AdvancementConfig {
   canShowTooltip_(event, pageRect) {
     let valid = true;
     let tagName;
+    const target = dev().assertElement(event.target);
 
     if (this.isInScreenSideEdge_(event, pageRect)) {
       return false;
     }
 
+    if (matches(target, AFFILIATE_LINK_SELECTOR)) {
```

Comment:
Can we fix this selector instead? https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story-embedded-component.js#L74

If we make sure that this selector doesn't match your affiliate link in the first place, it sounds a bit safer since it'd fix any current or future usage of that "embedded components tooltip".